src/change_import_style.ts

```diff
@@ -74,15 +74,20 @@ function processCommonJs(args: any, initialContents: string): string {
     }
 
     const exportSymbols = symbols.reduce((currentSymbols, symbol) => {
-      return currentSymbols + `exports.${symbol[0]} = ${symbol[1]};\n`;
+      return currentSymbols + `export const ${symbol[0]} = ${symbol[1]};\n`;
```

Comment:
Instead of using `export const ...`, can we use `export { X as Y};`? It avoids any possible problems if the symbol was already defined as a variable:

```js
const Proto = 0;

export const Proto = Proto;
```

The `export-as` syntax also seems to fit this use case better since we're renaming things, not defining them.

https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export

Reply:
I attempted to use the `export-as` syntax, but wasn't able to make it work due to the way the protos are defined in a nested structure.

This example throws a syntax error:
```javascript
export { proto.test.bazel.proto.Pizza as Pizza };
```

There are a couple ways we could probably make it work if we needed to. Is the example you provided something you've encountered? Short of an export being named `goog` or `global`, I don't see a possibility for name conflicts.

Nevertheless, while experimenting with `export-as` I managed the improve the existing export to:
```javascript
export const { Pizza, PizzaSize } = proto.test.bazel.proto
```